src/subdomains/generic/user/user.module.ts

```diff
@@ -83,7 +84,17 @@ import { IpLogRepository } from './models/ip-log/ip-log.repository';
     LinkService,
     WebhookService,
     LimitRequestNotificationService,
+    GeoLocationService,
+  ],
+  exports: [
+    UserService,
+    UserDataService,
+    KycService,
+    SpiderService,
+    SpiderApiService,
+    LinkService,
+    WebhookService,
+    GeoLocationService,
```

Comment:
`GeoLocationService` should not be provided and exported here, but exported from other module